### PR TITLE
CATALOG-9785: [update] Catalog, Category, Category Trees

### PR DESCRIPTION
**Pull request by NataliiaNefodova**
# [CATALOG-9785]

## What changed?
* Update swagger file for Categories, Category Trees

## Release notes draft
* Update swagger file for Categories, Category Trees

ping @NataliiaNefodova 

[CATALOG-9785]: https://bigcommercecloud.atlassian.net/browse/CATALOG-9785?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by olysenko — DISMISSED** [review]

File: reference/catalog/category-trees_catalog.v3.yml [hunk 1]
```diff
@@ -926,8 +936,102 @@ components:
...
+          description: The error title describing the particular error.
+        type:
+          type: string
+        errors:
+          title: Detailed Errors
+          type: object
+          properties: { }
+          additionalProperties: true
+  responses:
+    NotFoundError:
```
Comment (bc-traciporter) [inline reference/catalog/category-trees_catalog.v3.yml:1000]: Why is this schema is needed? It is only listed under schemas and responses but is not called in any of the endpoints.

**Review by bc-traciporter — COMMENTED** [review]
I made one comment.

File: reference/catalog/category-trees_catalog.v3.yml (hunk 1 above)
Comment (NataliiaNefodova) [reply reference/catalog/category-trees_catalog.v3.yml]: Removed

**Review by mykhailo-banin — APPROVED** [review]
🟢 LGTM

**Pull request merged**